### PR TITLE
plot weights in logistic regression

### PR DESCRIPTION
**Pull request by pascoesean**
investigating weights of each component in the 40 component model using logistic regression. Target variable can be modified to see how components are related to variance correlated with any of the categories in the observations anndata object

### CODE REVIEW

File: sccp/figures/figureS3b.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            .drop_duplicates()
+        )
+
+    group_labs = status.set_index('sample_ID')[group_to_predict]
+
+    _, factors, _, = openPf2(rank = rank, dataName = 'lupus', optProjs=True)
+        
+    A_matrix = factors[0]
+
+    contribs = getCompContribs(A_matrix, group_labs.to_numpy(), penalty_amt= 50)
```
Comment (andrewram4287) [inline sccp/figures/figureS3b.py:46]: @pascoesean is the penalty amount obtained from figureS3c.py??

**Comment by andrewram4287** [comment]
@pascoesean This all looks good to me! The only thing is to remove the old .npy files you committed before!

**Review by aarmey — APPROVED** [review]
This all looks fine, assuming it builds.

File: sccp/figures/figureS3b.py (hunk 1 above)
Comment (pascoesean) [reply sccp/figures/figureS3b.py:46]: @andrewram4287 yup!

Comment (pascoesean) [reply sccp/figures/figureS3b.py:46]: figureS3_cv-batch_ 40, 41, 42, 43, 44, 45

**Pull request merged**